src/lib/QueueLib.sol

```diff
@@ -136,7 +136,7 @@ library QueueLib {
         Queue storage self,
         mapping(uint256 => NodeOperator) storage nodeOperators,
         uint256 maxItems
-    ) external returns (uint256 toRemove) {
+    ) external returns (uint256 toRemove, uint256 toVisit) {
```

Comment:
```suggestion
    ) external returns (uint256 removed, uint256 lastRemovedBatch) {
```

Reply:
It's not a batch, it's rather the batch's 1-based index.

Reply:
`lastRemovedAtDepth` maybe?